modules/adblocker/sources/manager.es

```diff
@@ -155,9 +170,14 @@ export default class EngineManager {
     // As a last resort, we initialize an empty instance of the engine. It will
     // then be updated by parsing full-versions of the lists.
     if (this.engine === null) {
-      this.engine = new AdblockerLib.WebExtensionBlocker();
+      this.engine = new AdblockerLib.WebExtensionBlocker({
```

Comment:
This is only used as a fallback when we could not fetch the serialized engine from cache or CDN. It should almost never reach this point.

Reply:
I see. If the adblocker changes (in the other project) would be merged, then the CDN would be updated. In local testing, we currently hit it because the (local) adblocker engine is newer. But it would not work in a production setting.

We could make the flag opt-out for Firefox. There is already a fallback if the extension would not have the new API permission (webNavigation). So, it would not break other clients.